PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+[comment]: # (NOTE: Please use the following prefixes for the PR subject field)
```

Comment:
This is a will appear in the template but will not appear in the posted PR body even if left undeleted.